swift-quiz.playground/Contents.swift

```diff
@@ -1,5 +1,48 @@
 //: Playground - noun: a place where people can play
 
-import UIKit
+class Quiz {
+    let result:Int = 0
+    
+    let arrQuestions:[Question] = []
+    
+    func addQuestions(questions: [AnyObject]) {
```

Comment:
I think Quiz object must have a constructor with 1 json string containing the entire data structure.